### PR TITLE
[Quota] Add new commands `az quota request list/show` to support managing quota request

### PR DESCRIPTION
**Pull request by Jing-song**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

AAZ model https://github.com/Azure/aaz/pull/346
Since the quota [get api](https://learn.microsoft.com/en-us/rest/api/quota/quota/get?view=rest-quota-2021-03-15-preview&tabs=HTTP) does not return `provisioning state`, the `az quota wait` command is invalid, so remove it.
### Related command

`quota create/update/list/show`
`quota usage list/show`
`quota request list/show`

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Quota

File: src/quota/azext_quota/generated/action.py [hunk 1]
```diff
@@ -7,50 +7,3 @@
...
-            v = properties[k]
-
-            if kl == 'value':
-                d['value'] = v[0]
-
-            elif kl == 'limit-type':
-                d['limit_type'] = v[0]
-
-            elif kl == 'limit-object-type':
-                d['limit_object_type'] = v[0]
```
[the remark this answers is not in the document]
Comment (Jing-song) [reply src/quota/azext_quota/generated/action.py:48]: It was migrated to aaz's logic.
https://github.com/Azure/azure-cli-extensions/pull/7005/files#diff-339f3d8af7a01232646c3af83b5b87936b92f5c7f6fcfa4c848997fec67dbb99R228-R231

File: src/quota/azext_quota/aaz/latest/quota/_create.py [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+        _args_schema.scope = AAZStrArg(
+            options=["--scope"],
+            help="The target azure resource URI.",
+            required=True,
+        )
+        _args_schema.properties_org = AAZObjectArg(
+            options=["--properties-org"],
+            help="Additional properties for the specific resource provider.",
+            blank={},
+        )
```
Comment (kairu-ms) [inline src/quota/azext_quota/aaz/latest/quota/_create.py:71]: Did you try to generate freeform dict directly?

File: src/quota/azext_quota/generated/_params.py [hunk 3]
```diff
@@ -10,49 +10,9 @@
...
-        c.argument('scope', type=str, help='The target Azure resource URI.')
-
-    with self.argument_context('quota create') as c:
-        c.argument('resource_name', type=str, help='The resource name for a given resource provider.')
-        c.argument('scope', type=str, help='The target Azure resource URI.')
-        c.argument('limit_object', action=AddLimitobject, nargs='+', help='The resource quota limit value.',
-                   arg_group='Limit')
-        c.argument('name', type=str, help=' The resource type name.')
-        c.argument('resource_type', type=str, help='The resource type name.')
-        c.argument('properties', type=validate_file_or_dict, help='The additional properties for the specific resource provider.')
```
[the remark this answers is not in the document]
Comment (Jing-song) [reply src/quota/azext_quota/generated/_params.py:40]: Yes, it has been added
https://github.com/Azure/azure-cli-extensions/pull/7005/files#diff-05e9dff9abfc9caff7f99897d85d535dbf04a44bc5a8992d540a549835136dcfR52-R56

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**